lib/spec/change.spec.ts

```diff
@@ -356,6 +358,20 @@ describe('Change Tracking', () => {
 
 				expect(changeCount).to.eq(1);
 			})
+
+			it("ignores many-to-many insert for untracked entity", async () => {
+				const entity = await unTrackedFactory.create();
+				const related = await relatedFactory.createMany(3);
+
+				// This inserts records into the join table implcitly created for MTM
+				// relationships. These should not be tracked, and should not fail either.
```

Comment:
Should not fail should be expressed in test rather than comment. Probably we should use `chai-as-promised` here and do something like:

```typescript
await expect(entity.save()).to.be.fulfilled;
```